src/storage-client/src/storage_collections.rs

```diff
@@ -1601,17 +1614,20 @@ where
                     self_collections.insert(id, collection_state);
                 }
                 DataSource::Other(DataSourceOther::TableWrites) => {
-                    let register_ts = register_ts
```

Comment:
@jkosh44 we're trading one hack for a different hack... 🙈

Reply:
I feel better about this one. The `txn` upper _is_ the logical upper of all tables. `register_ts` was just some random timestamp that we were pretty sure would eventually be used for a write to the table.